encord/client.py

```diff
@@ -793,6 +795,13 @@ def get_project(self, include_labels_metadata=True) -> OrmProject:
             OrmProject, payload={"include_labels_metadata": include_labels_metadata}, retryable=True
         )
 
+    def get_project_v2(self) -> ProjectOrmV2:
```

Comment:
I'm not the one to usually suggest this, but an _ in the function name?

Reply:
Done!